### PR TITLE
Start 21w07a snapshot support

### PR DESCRIPTION
**Pull request by u9g**
I don't really understand how to include the `snapshot x` as a version so I just included the last number version like every other one from here https://wiki.vg/Protocol_version_numbers

### CODE REVIEW

File: data/pc/21w07a/protocol.json [hunk 1]
```diff
@@ -1918,12 +1918,15 @@
...
-              "name": "groundUp",
-              "type": "bool"
+              "name": "bitMaskLength",
+              "type": "varint"
             },
             {
-              "name": "ignoreOldData",
-              "type": "bool"
+              "name": "primaryBitMask",
+              "type": ["array", {
```
[the remark this answers is not in the document]
Comment (u9g) [reply data/pc/21w07a/protocol.json:1926]: talked about

File: data/pc/21w07a/protocol.json [hunk 2]
```diff
@@ -3920,6 +3944,37 @@
...
+              "type": "string"
+            },
+            {
+              "name": "destination",
+              "type": [
+                "switch",
+                {
+                  "compareTo": "destinationIdentifier",
+                  "fields": {
+                    "block": "i64",
```
Comment (rom1504) [inline data/pc/21w07a/protocol.json:3965]: this doesn't seem good
what is that long?

File: data/pc/21w07a/protocol.json [hunk 3]
```diff
@@ -3882,20 +3905,21 @@
...
+              "countType": "varint",
+              "type": ["container",
+              [
+                {
+                  "name": "tagType",
+                  "type": "string"
+                },
+                {
+                  "name": "tags",
+                  "type": "tags"
```
Comment (rom1504) [inline data/pc/21w07a/protocol.json:3919]: that is wrong

File: data/pc/21w07a/protocol.json (hunk 2 above)
Comment (u9g) [reply data/pc/21w07a/protocol.json:3965]: https://wiki.vg/Pre-release_protocol#Sculk_Vibration_Signal

Comment (rom1504) [reply data/pc/21w07a/protocol.json:3965]: yes wiki.vg is missing the information too.
I bet it's a position https://wiki.vg/Protocol#Data_types

File: data/pc/21w07a/protocol.json (hunk 3 above)
Comment (rom1504) [reply data/pc/21w07a/protocol.json:3919]: nevermind, misread, it's correct

**Review by rom1504 — APPROVED** [review]

File: data/pc/21w07a/protocol.json (hunk 2 above)
Comment (u9g) [reply data/pc/21w07a/protocol.json]: added

**Review by rom1504 — APPROVED** [review]

**Pull request merged**